src/locales/ja.json

```
@@ -0,0 +1,71 @@
+{
+ "All items were canceled from the order": "全てのアイテムが注文からキャンセルされました",
```

Hi,
Please convert the identation to 1 tab (2 space).
```suggestion
  "All items were canceled from the order": "全てのアイテムが注文からキャンセルされました",
```

---

@ymaheshwari1 
revised it. please check it.
thank you.

---

Looks good. Thanks for the contribution